core/primitives-core/src/config.rs

```diff
@@ -223,6 +223,17 @@ pub struct ExtCostsConfig {
     /// Cost of getting sha256 per byte
     pub keccak512_byte: Gas,
 
+    #[cfg(feature = "protocol_feature_math_extension")]
```

Comment:
Oh, another field under feature :(. Can we avoid that?

Reply:
Alt BN128 is also under a feature in the same vein?

Reply:
https://github.com/near/nearcore/pull/4380/commits/bffcbebcdbbb406c91115f0643f83ae4114b5bc4